src/token-group/generic-token-group.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we have rules/guidelines on when reusable sub-components should go into `./internal/components/[sub-component]` vs `./[component]/[sub-component]`? Unfortunately it seems we have a bit of a mix of both currently :/

Reply:
Typically we put internal components to under `/src/internal/components` but in this case it is a little controversial. In the file-upload we need to use sort of part of the `token-group` component, very similar how we use the popover internally. Instead of using the public popover in the annotation-context, breadcrumb-group, button-dropdown and chart-popover we use `./src/popover/container`. Both generic-token-group and popover-container are very tightly coupled to their public variants so I think keeping those closer is a good call.